src/crons/attendance-reminder-friday.ts

```diff
@@ -0,0 +1,75 @@
+import { ChannelType, type Client } from "discord.js";
+import { schedule } from "node-cron";
+
+import {
+    HACK_NIGHT_ATTENDANCE_CHANNEL_ID,
+    HACK_NIGHT_ATTENDANCE_ROLE_ID,
+} from "../utils/consts";
+
+// Every 15 Minutes starting at 9pm
+export default async function startTask(client: Client) {
+    const task = schedule("*/15 21-23 * * 5", handler(client));
+    return task.start();
+}
+
+function handler(client: Client) {
+    return async () => {
+        const channel = client.channels.cache.get(
+            HACK_NIGHT_ATTENDANCE_CHANNEL_ID,
+        );
+
+        if (!channel) {
+            console.error("Could not find channel");
+            return;
+        }
+
+        if (!channel.isSendable()) {
+            console.error("Cannot send messages to channel");
+            return;
+        }
+
+        if (channel.type !== ChannelType.GuildText) {
+            console.error("Cannot create threads in channel");
+            return;
+        }
+
+        const threads = await channel.threads.fetchActive();
+
+        if (!threads) {
+            console.error("Could not fetch active threads");
+            return;
+        }
+
+        const hackNightAttendanceThread = threads.threads
+            .filter((t) => {
+                return t.name.startsWith("");
```

Comment:
This matches all threads, is this intentional? If so probably can remove this filter haha